Source/textdat.cpp

```diff
@@ -782,6 +782,140 @@ const TextDataStruct alltext[] = {
+	    1, 5, TSFX_FARMER2A },
+	{ "Oh, I could use your help, but perhaps after you've saved the catacombs from the desecration of those beasts.|",
+	    1, 5, TSFX_FARMER6 },
+	{ "I need something done, but I couldn't impose on a perfect stranger. Perhaps after you've been here a while I might feel more comfortable asking a favor.|",
+	    1, 5, TSFX_FARMER7 },
+	{ "I see in you the potential for greatness.  Perhaps sometime while you are fulfilling your destiny, you could stop by and do a little favor for me?|",
+	    1, 5, TSFX_FARMER8 },
+	{ "I think you could probably help me, but perhaps after you've gotten a little more powerful. I wouldn't want to injure the village's only chance to destroy the menace in the church!|",
+	    1, 5, TSFX_FARMER9 },
+	{ "Me, I'm a self-made cow.  Make something of yourself, and... then we'll talk.|",
+	    1, 5, TSFX_COWSUT10 },
+	{ "I don't have to explain myself to every tourist that walks by!  Don't you have some monsters to kill?  Maybe we'll talk later.  If you live...|",
+	    1, 5, TSFX_COWSUT11 },
+	{ "Quit bugging me.  I'm looking for someone really heroic.  And you're not it.  I can't trust you, you're going to get eaten by monsters any day now... I need someone who's an experienced hero.|",
+	    1, 5, TSFX_COWSUT12 },
+	{ "All right, I'll cut the bull.  I didn't mean to steer you wrong.  I was sitting at home, feeling moo-dy, when things got really un-stable; a whole stampede of monsters came out of the floor!  I just cowed.  I just happened to be wearing this Jersey when I ran out the door, and now I look udderly ridiculous.  If only I had something normal to wear, it wouldn't be so bad.  Hey!  Can you go back to my place and get my suit for me?  The brown one, not the gray one, that's for evening wear.  I'd do it myself, but I don't want anyone seeing me like this.  Here, take this, you might need it... to kill those things that have overgrown everything.  You can't miss my house, it's just south of the fork in the river... you know... the one with the overgrown vegetable garden.|",
+	    1, 5, TSFX_COWSUT4A },
+	{ "Cloudy and cooler today.  Casting the nets of necromancy across the void landed two new subspecies of flying horror; a good day's work.  Must remember to order some more bat guano and black candles from Adria; I'm running a bit low.|",
+	    1, 5, USFX_SKLJRN1 },
+	{ "I have tried spells, threats, abjuration and bargaining with this foul creature -- to no avail.  My methods of enslaving lesser demons seem to have no effect on this fearsome beast.|",
+	    1, 5, PS_NARATR6 },
+	{ "My home is slowly becoming corrupted by the vileness of this unwanted prisoner.  The crypts are\tfull of shadows that move just beyond the corners of my vision.  The faint scrabble of claws dances at the edges of my hearing. They are searching, I think, for this journal.|",

```

Comment:
what is \t doing here? : P

Reply:
Making sure there is a tab, do we use a literal tab elsewhere?

Reply:
I don't think so, that's why I asked ;p